server/plugin/http.go

```diff
@@ -69,7 +69,16 @@ func (c *client) Call(method, path, contentType string, inBody io.Reader, out in
 		if strings.Contains(err.Error(), "invalid character '<'") {
 			return nil, http.StatusInternalServerError, ErrorContentTypeNotJSON
 		}
-		return nil, http.StatusInternalServerError, err
+
+		updatedError := strings.ReplaceAll(err.Error(), c.plugin.getConfiguration().MattermostSiteURL, "")
```

Comment:
```suggestion
		updatedError := strings.ReplaceAll(err.Error(), c.plugin.getConfiguration().ServiceNowBaseURL, "")
```
We have to replace the ServiceNow URL as we are making request to that URL.